### ISSUE TITLE
Enable specifying the required message for "SliderElement" component

### ISSUE THREAD

**Issue by benjaminpochat** [dohomi/react-hook-form-mui#234]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [x] I have tested the latest version

### Summary 💡

On several component such as TextFieldElement or SelectElement, its possible to define a specific label to warn a user that a field is required with the property "validation". For instance :

```
                <TextFieldElement
                    name="aTextField"
                    validation={{ required: 'Champ obligatoire' }}/>
```

Unfortunately, it not possible with the slider component.
The following code does not work

```
               <SliderElement
                    name="aTextField"
                    validation={{ required: 'Champ obligatoire' }}/>
```

It would be great to enable this feature for sliders as well.

As far as I understood, there is no workaround.

**Comment by sadik-malik** [comment]
@benjaminpochat try rules prop instead of validation.

```jsx
 rules={{ required: 'Champ obligatoire' }}
```

@dohomi we may need to either deprecate the rules prop or validation prop, because having two prop doing the same thing is confusing.

**Comment by dohomi** [comment]
I think this is a great candidate for the v7. I wanted to do that already but didn't want to introduce breaking changes. v7 would be a great fit to get rid of the duplication

**Comment by benjaminpochat** [comment]
Using prop instead of validation solves the problem.
It would be great if the configuration could be homogeneous in a next version, maybe by supporting both for a while, documenting one of both as deprecated in the doc until the version it will be completely removed.

Thanks for this useful lib.

**Comment by dohomi** [comment]
fixed in v7.0.0-beta.2
